internal/cron-jobs/handler.go

```diff
@@ -4,9 +4,9 @@ import (
 	"net/http"
 
 	"github.com/gorilla/mux"
-	errors "github.com/skobelina/currency_converter/utils/errors"
-	"github.com/skobelina/currency_converter/utils/rest"
-	"github.com/skobelina/currency_converter/utils/serializer"
+	errors "github.com/skobelina/currency_converter/pkg/utils/errors"
+	"github.com/skobelina/currency_converter/pkg/utils/rest"
```

Comment:
https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common pkg already tell us about general porpuses package we can leave just /pkg/errors